mfrc522/iso14443com.py

```diff
@@ -0,0 +1,55 @@
+import crcmod
+from .mfrc522 import *
+import struct
+
+SEL_CASCADE_1   = 0x93
```

Comment:
IMHO an enum would be nicer. ("Namespaces are one honking great idea -- let's do more of those!
")